components/atom/input/src/index.js

```diff
@@ -10,6 +10,7 @@ import InputWrapper from './InputWrapper'
 const NATIVE_TYPES = ['text', 'number', 'date']
 
 const changeHandler = (ev, cb) => {
+  // eslint-disable-next-line
```

Comment:
You should not disable rules until it's necessary.

Reply:
Sorry, I told him to do it so he could commit while I'm looking why the linter is failing